setup.py

```diff
@@ -35,7 +35,7 @@
         'esgf-pyclient',
         # pin requests_cache: https://github.com/ESGF/esgf-pyclient/issues/71
         'requests_cache<0.6',
-        'esmpy!=8.1.0',  # see github.com/ESMValGroup/ESMValCore/issues/1208
+        'esmpy',
```

Comment:
Why are you changing this file?

Reply:
I should not, I need to make sure it's in line with `main`

Reply:
re-aligned env to `main`